mmf/models/vit.py

```diff
@@ -0,0 +1,486 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
I think we will probably delegate on HuggingFace to provide ViT implementation. Can you create a folder  `mmf/models/albef` and put this there. Instead of registering ViT encoder in `mmf/modules/encoders.py`, you can register the encoder `mmf/models/albef/vit.py` itself as `albef_vit_encoder` to be very specific.

Reply:
Hi @apsdehal thank you for your feedback. I will implement the requested changes.